x-pack/plugins/security_solution/public/resolver/view/panels/node_events_of_type.tsx

```diff
@@ -44,29 +51,56 @@ export const NodeEventsInCategory = memo(function ({
   );
   const events = useSelector((state: ResolverState) => selectors.nodeEventsInCategory(state));
 
+  const isLoading = useSelector(selectors.isLoadingNodeEventsInCategory);
+  const hasError = useSelector(selectors.hadErrorLoadingNodeEventsInCategory);
   return (
     <>
-      {eventCount === undefined || processEvent === null ? (
+      {isLoading || processEvent === null ? (
         <StyledPanel>
           <PanelLoading />
         </StyledPanel>
       ) : (
         <StyledPanel data-test-subj="resolver:panel:events-in-category">
-          <NodeEventsInCategoryBreadcrumbs
-            nodeName={eventModel.processNameSafeVersion(processEvent)}
-            eventCategory={eventCategory}
-            eventCount={eventCount}
-            nodeID={nodeID}
-            eventsInCategoryCount={eventsInCategoryCount}
-          />
-          <EuiSpacer size="l" />
-          <NodeEventList eventCategory={eventCategory} nodeID={nodeID} events={events} />
+          {hasError ? (
+            <EuiCallOut
```

Comment:
Maybe we migrate this to `panel_content_error` and expand from this to `error, 404, and loading` component to be used in the graph itself and the panel like @oatkiller suggested a while back